modules/social_features/social_event/social_event.module

```diff
@@ -270,6 +271,17 @@ function social_event_views_query_alter(ViewExecutable $view, QueryPluginBase $q
   }
 }
 
+/**
+ * Implements hook_entity_insert().
+ */
+function social_event_entity_insert(EntityInterface $entity): void {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if ($entity instanceof EventEnrollment && $route_name !== 'social_event_managers.add_enrollees') {
```

Comment:
Do we need this condition here? 
I think we need to prevent requests/invite, but not direct add by event managers.
In any case - please add a comment here. Thanks!

Reply:
Nope, I have updated the condition.